addons/l10n_cn/views/account_move_view.xml

```diff
@@ -1,6 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
+        <record id="view_invoice_tree_inherit_i10n_cn" model="ir.ui.view">
+            <field name="name">view.invoice.tree.inherit.i10n.cn</field>
+            <field name="model">account.move</field>
+            <field name="inherit_id" ref="account.view_invoice_tree"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='ref']" position="after">
+                    <field name="fapiao" optional="hide" invisible="country_code != 'CN' or move_type not in ['out_invoice', 'out_refund', 'in_invoice', 'in_refund']"/>
```

Comment:
Not sure that the invisible makes much sense as the field will be empty anyways in these cases...  @alialfie (in the list view, the best thing we have is still the optional hide, I am afraid)

Reply:
@tanghulu0608 As Josse mentioned, I was incorrect here to suggest the invisible, my bad. Could you please remove the `invisible` attribute again and we can merge? :) Thanks!

Reply:
@alialfie  `invisible` attribute removed.